Cargo.toml

```diff
@@ -13,7 +13,7 @@ license = "MIT"
 anyhow = "1.0.70"
 async-trait = "0.1.68"
 deno_semver = "0.5.4"
-deno_lockfile = "0.20.0"
+deno_lockfile = { git = "https://github.com/zebreus/deno_lockfile.git", branch = "work-on-lockfile" }
```

Comment:
it seems like simply updating to [v0.22](https://github.com/denoland/deno_lockfile/releases) would default to lockfile v4.